From 264dbeaaf6c38de41f2591d8772ec648e15638aa Mon Sep 17 00:00:00 2001
From: Jared Morrow <jm@joyent.com>
Date: Mon, 11 Sep 2017 13:24:51 -0600
Subject: [PATCH] MANTA-3357 manta-scripts update script repos.sh no longer
 works with modern git Reviewed by: Kody A Kantor <kody.kantor@gmail.com>
 Approved by: Kody A Kantor <kody.kantor@gmail.com>

---
 update/repos.sh | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/update/repos.sh b/update/repos.sh
index fd24a20..b8763e5 100755
--- a/update/repos.sh
+++ b/update/repos.sh
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2017, Joyent, Inc.
 #
 
 ###############################################################################
@@ -19,6 +19,8 @@
 # with repos, you can fix and safely rerun.
 ###############################################################################
 
+set -o xtrace
+
 REPOS=(
     binder
     electric-moray
@@ -72,7 +74,7 @@ for repo in "${REPOS[@]}"; do
         continue
     fi
 
-    REPO_GIT_SHA=$(git submodule $DEP_LOC | cut -c 2-41)
+    REPO_GIT_SHA=$(git submodule status $DEP_LOC | cut -c 2-41)
     if [ "$MS_GIT_SHA" == "$REPO_GIT_SHA" ]; then
         echo "$repo already has latest manta-scripts.  Not updating."
         continue
-- 
2.21.0

